src/main/webapp/profile.html

```diff
@@ -7,18 +7,28 @@
     <script src="script.js"></script>
   </head>
 
-  <body onload="getProfile()"> 
-    <h2>Profile</h2>
-    <p id="saved-profile-status"></p>
-    <form action="javascript:postProfile()" method="POST" onchange="clearSavedProfileStatus()">
-      <span>Username</span>
-      <textarea id="name-entry" required="true" placeholder="Place your username here..." name="name-input"></textarea>
-      <br/><input id="vegetarian-checkbox" type="checkbox"/><span>Vegetarian</span>
-      <br/><input id="vegan-checkbox" type="checkbox"/><span>Vegan</span>
-      <br/><input id="gluten-checkbox" type="checkbox"/><span>Gluten Free</span>
-      <br/><input id="dairy-checkbox" type="checkbox"/><span>Dairy Free</span>
-      <br/><span>Allergies</span><textarea id="allergies-entry" placeholder="Separate allergies using commas.." name="allergy-input"></textarea>
-      <br/><input id="profile-submit-button" type="submit" />
-    </form>
+  <body id="profile-background" onload="getProfile()"> 
+    <div class="content">
+      <h2 id="profile-header">Profile</h2>
+      <form class="content" id="profile-form" action="javascript:postProfile()" method="POST" onchange="clearSavedProfileStatus()">
```

Comment:
I don't think you need the "javascript" added. Can just put the function name

Reply:
I tried removing the javascript, it gave me an error that it could not find /postProfile().